### DISCUSSION TITLE
Collecation editor: On question preview all questions are visible

### DISCUSSION THREAD

**Discussion by manojSRawat** [Sunbird-inQuiry/Community#30] [Bugs]
In release-5.0.0 when we are trying to open question detail all questions are visible in the QUML player.

<img width="1124" alt="image" src="https://user-images.githubusercontent.com/22637774/186140056-416274cd-5f32-46ca-915f-dd12a6519d6c.png">

Expected 
<img width="1238" alt="image" src="https://user-images.githubusercontent.com/22637774/186175268-abaf72cc-fad9-4f2c-959a-97dbe3c9c71c.png">

cc: @pallakartheekreddy @tarang27 @surendrasinghs @prerna789

**Comment by rajnishdargan** [comment]
@manojSRawat
**Player config for Questionset Preview:**
playerconfig: {
    "context": {...},
    "config": {...},
    "metadata": {...} // complete questionset preview, metadata of questionset should be passed.
    "data": {...}
}

**Player config for single question preview:**
playerconfig: {
    "context": {...},
    "config": {...},
    "metadata": {...} // single question preview, metadata of question should be passed.
    "data": {...}
}
@manojSRawat In single question preview you are passing metadata of questionset, metadata should have question metadata when you are trying to preview a single question.

cc: @pallakartheekreddy @surendrasinghs @Prerna789 @tarang27 @itsvick

**Comment by pallakartheekreddy** [comment]
@manojSRawat is @rajnishdargan response answered your query??

**Reply by manojSRawat** [reply]
@pallakartheekreddy we've already raised a fix for this and it's working fine staging.
